src/localization/locales/fi.json

```
@@ -183,6 +193,11 @@
       "clickToPreview": "Klikkaa esikatsellaksesi",
       "addButton": "Lisää liite",
       "uploadDialogTitle": "Lataa uusi liite",
+      "uploadExistingFileConfirmationDialog": {
+        "title": "Lataa olemassaoleva liite",
+        "description": "Tiedosto, jota yrität ladata, on jo olemassa palvelimen tallennustilassa. Sitä käytetään uudelleen tähän tehtävään. Haluatko varmasti jatkaa?",
```

Related to the english suggestion
```suggestion
        "description": "Samanniminen tiedosto, jota yrität ladata on jo olemassa. Haluatko käyttää olemassa olevaa tiedostoa uudelleen tässä tehtävässä? Voit myös muuttaa lataamasi tiedoston nimeä ja ladata sen uudelleen.",
```